templates/tfengine/components/cicd/configs/run.sh

```diff
@@ -0,0 +1,49 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#!/bin/bash
+set -e
+set -x
+
+DIRS=(
+  {{- range .managed_modules}}
+  {{.}}
+  {{- end}}
+)
+
+ACTION="plan"
+ROOT=""
+
+while getopts 'a:r:' c
+do
+  case $c in
+    a) ACTION=${OPTARG} ;;
+    r) ROOT=${OPTARG} ;;
+    *)
+      echo "Invalid flag ${OPTARG}"
+      exit 1
+      ;;
+  esac
+done
+
+for d in ${DIRS[@]}
+do
```

Comment:
Suggest: The loop right now only works with absolute paths and breaks with relative paths. Consider either running `pushd` at the beginning and `popd` after, or converting ROOT to an absolute path with `readlink`.

Reply:
I tested this with ROOT="../../" earlier and it worked fine. Do you know a case that breaks?

Reply:
Good catch, let me fix this